web-new/src/app/globals.css

```diff
@@ -5,10 +5,36 @@
 @layer base {
   body {
     @apply bg-brand-80 text-brand-20;
-    @apply font-secondary;
+    @apply font-secondary text-body-1;
     box-sizing: border-box;
     background-image: url("../../public/images/bg-body.png");
     background-size: cover;
     background-repeat: no-repeat;
+    background-attachment: fixed;
+  }
+}
+
+
+@layer components {
+  .text-heading-1 {
+    @apply font-primary text-[28px] leading-[30px];
```

Comment:
We should not use hardcoded values like this, instead they should go in the tailwind config

Reply:
https://github.com/darklakefi/darklake-monorepo/pull/255#discussion_r1969732720